axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java

```diff
@@ -246,7 +246,7 @@ protected void removeOneMove(Move move, ActionResponse response) throws Exceptio
   public void deleteMultipleMoves(ActionRequest request, ActionResponse response) {
     try {
       List<Long> moveIds = (List<Long>) request.getContext().get("_ids");
-      if (!moveIds.isEmpty()) {
+      if (moveIds != null && !moveIds.isEmpty()) {
```

Comment:
Better to use CollectionUtils.isEmpty of org.apache.commons.collections , it has a null-safe check: 

`if (CollectionUtils.isEmpty(moveIds) {`